Swap.cpp

```diff
@@ -0,0 +1,55 @@
+#include<iostream>
+using namespace std;
+
+void SwapRef(int &a, int &b) 
+{
+   int temp;
+   temp = a; 
+   a = b;    
+   b = temp; 
+     
+   return;
+}
+
+void SwapPoint(int *a, int *b)
+{
+
+   int temp;
+   temp = *a;   
+   *a = *b;     
+   *b = temp;   
+  
+   return;
+}
+
+int Func (int A[],int n)
+{
+	int i;
+	for (i=0;i<n;i++)
```

Comment:
For better understanding use opening and closing brackets when you use for loop. Even if its just one line inside the loop.